cpp/include/cudf/detail/reverse.hpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2019-2021, NVIDIA CORPORATION.
```

Comment:
New file, so copyright just current year.  I think you are not obliged to use NVIDIA's copyright, just the rest of the license.
```suggestion
 * Copyright (c) 2021, NVIDIA CORPORATION.
```